docs/ethical-advertising.rst

```diff
@@ -0,0 +1,123 @@
+Ethical Advertising
+===================
+
+Read the Docs is a large free website on the internet.
+There is one proven business model to support this kind of site:
+Advertising.
+
+We talk a bit :ref:`below <ethical-info>` about our philosophy on advertising.
+
+If you're a marketer,
+we have :ref:`information <ethical-buy-ads>` on how to get in touch with us about running ads.
+
+.. _ethical-info:
+
+Our worldview
+-------------
+
+We're doing advertising properly:
```

Comment:
It makes it less impactful. If we want it to be a powerful document, talking in the passive voice is not a great way to accomplish it.